c_models/src/my_models/implementations/my_full_neuron_impl.h

```diff
@@ -91,6 +91,14 @@ static bool neuron_impl_do_timestep_update(
     return false;
 }
 
+__attribute__((unused)) // Marked unused as only used sometimes
+static void neuron_impl_record_structural(
+		uint32_t neuron_index, uint32_t rec_value) {
+	// See the standard implementation for how to turn this on if needed
+	use(neuron_index);
+	use(rec_value);
+}
+
```

Comment:
Seems surprising that you have to do this here when other recordings don't need this... Isn't this managed by the neuron_recording file?

Reply:
I have a feeling that it has to pass through here in order to reach the correct place to actually perform the recording.  But it may be that after the reconfiguration in sPyNNaker that this becomes unnecessary - we shall see.

Reply:
This isn't necessary any more - in fact the reconfiguration means this PR is likely now unnecessary and can probably be closed.